snmp/CHANGELOG.md

```diff
@@ -2,10 +2,11 @@
 
 ## Unreleased
 
-## 6.1.0 / 2023-08-10
+## 6.1.0 / 2023-08-16
 
 ***Added***:
 
+* Add hardware metrics for F5 (non-categorical) ([#15568](https://github.com/DataDog/integrations-core/pull/15568))
```

Comment:
Sorry if this is confusing, but you're supposed to add this line to the `## Unreleased`, 6.1.0 being already release. `ddev release changelog new` will do the change for you. More info https://datadoghq.dev/integrations-core/guidelines/pr/#changelog-entries